docs/MapColonies/vector/Services/valhalla_service.md

```diff
@@ -0,0 +1,63 @@
+---
+id: vector-valhalla-service
+slug: valhalla
+title: Valhalla 🧭
+description: Directions and isochrones by OSM
+tags:
+  - vector
+  - API
+  - osm
+---
+
+# Valhalla 🧭
+
+Valhalla is an open source routing engine and accompanying libraries for use with OpenStreetMap data. Valhalla also includes tools like time+distance matrix computation, isochrones, elevation sampling, map matching and tour optimization (Travelling Salesman).
```

Comment:
We should add a link to the `Valhalla` docs and website to credit them.
Same for `Nominatim`.